app/helpers/prison_admin_helper.rb

```diff
@@ -0,0 +1,5 @@
+module PrisonAdminHelper
+  def reason_request_cannot_be_processed(booking_response)
+    content_tag(:h2, t(".#{booking_response.processing_state_name}"))
```

Comment:
This method seems out of place to me - especially moving the H2 out of the template.